hosted/gke/helper/helper_cluster.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 // CreateGKEHostedCluster creates the GKE cluster
-func CreateGKEHostedCluster(client *rancher.Client, displayName, cloudCredentialID, k8sVersion, zone, project string, npSize int) (*management.Cluster, error) {
+func CreateGKEHostedCluster(client *rancher.Client, displayName, cloudCredentialID, k8sVersion, zone, project string, updateFunc func(clusterConfig *gke.ClusterConfig)) (*management.Cluster, error) {
```

Comment:
Nice addition of updateFunc !
Can we make it an optional parameter ?

```suggestion
func CreateGKEHostedCluster(client *rancher.Client, displayName, cloudCredentialID, k8sVersion, zone, project string, updateFunc ...func(clusterConfig *gke.ClusterConfig)) (*management.Cluster, error) {
```